grc/satellites_check_address.block.yml

```diff
@@ -28,6 +32,18 @@ outputs:
 
 templates:
     imports: import satellites
-    make: satellites.check_address(${address}, ${direction})
+    make: satellites.check_address(${address}, ${direction}, ${digicallsign})
+
+documentation: |-
+    Enter an address (choose "from" for source or "to" for destination callsign) and/or enter a digipeater callsign.
+    The address callsign may have a SSID, the digipeater callsign must not have a SSID.
```

Comment:
Can you describe accurately which frames are passed to the `ok` output? At the moment nothing in this documentation gives a clue as to whether the `or` in line 79 is an `or` or an `and`.